backend/nouns/DeleteNouns.ts

```diff
@@ -1,15 +1,34 @@
 import { sequelize } from "../DB.js";
 import { router } from "../Router.js";
+import { param, validationResult } from "express-validator";
+import {
+  invalidRequest,
+  notFound,
+  successNoContent,
+} from "../Utils/EndpointResponses.js";
 
-router.delete("/api/nouns/:nounId", async (req, res) => {
-  const numDeleted = await sequelize.models.Noun.destroy({
-    where: {
-      id: req.params.nounId,
-    },
-  });
-  if (numDeleted === 0) {
-    res.sendStatus(404);
-  } else {
-    res.sendStatus(200);
+router.delete<Params>(
+  "/api/nouns/:nounId",
+  param("nounId").isInt().toInt(),
+  async (req, res) => {
+    const validationErrors = validationResult(req);
+    if (!validationErrors.isEmpty()) {
+      return invalidRequest(res, validationErrors);
+    }
+
+    const numDeleted = await sequelize.models.Noun.destroy({
+      where: {
+        id: req.params.nounId,
+      },
+    });
+    if (numDeleted === 0) {
+      notFound(res, `No such noun with id '${req.params.nounId}'`);
```

Comment:
Is it helpful to include the context in the response message? Such as, `No such noun with id '${req.params.nounId}'. Could not delete noun.`. Or is that typically too verbose?